example-get-started-experiments/code/.github/workflows/run-experiment.yml

```
@@ -64,32 +65,27 @@ jobs:
           STUDIO_TOKEN: ${{ secrets.STUDIO_TOKEN }}
           DVCLIVE_LOGLEVEL: DEBUG
         run: |
-          dvc pull
           cml ci --fetch-depth 0
+          dvc pull
           dvc exp run ${{ github.event.inputs.exp-run-args }}
           dvc remote add --local push_remote s3://dvc-public/remote/get-started-pools 
-          dvc push -r push_remote
-  
-      - name: Report
-        env:
-          REPO_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
-        run: |
-          echo "## Metrics" > report.md
-          dvc metrics diff main --md >> report.md
-          echo "## Params" >> report.md
-          dvc params diff main --md >> report.md
 
-      - name: Workflow Dispatch PR
+      - name: Workflow Dispatch Sharing
         if: github.event_name == 'workflow_dispatch'
         env:
-          REPO_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+          STUDIO_TOKEN: ${{ secrets.STUDIO_TOKEN }}
         run: |
-          cml pr --skip-ci .
+          dvc exp push origin --rev HEAD -r push_remote
```

```suggestion
          dvc exp push origin -r push_remote
```

Can we drop it now? By the way, not sure why we weren't recommending the simpler `-A` here anyway.

---

Yep! was also not released when I pushed this